docs/user/installation-in-wordpress-multisite.md

```diff
@@ -0,0 +1,109 @@
+# BuddyPress Installation Guide for WordPress Multisite
+
+Welcome to the BuddyPress installation guide for WordPress Multisite! This document is designed to help beginners install and configure BuddyPress on their WordPress Multisite network. Whether you're setting up a community site, a social network, or a collaborative space, BuddyPress is a powerful plugin that adds social networking features to your WordPress site.
+
+## Before You Begin
+
```

Comment:
```suggestion

Just like there's only one common Users database table shared across your multisite network, BuddyPress only installs a common set a database tables for your network. This means, out of the box, BuddyPress does not provide a way to have more than one instance of it into your network so that you could have different communities for each of your network sites. In other words, activating BuddyPress network widely will only power your network with one BuddyPress instance you can choose to house on the main or a secondary site of your network.
```